Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -108,6 +110,75 @@ - (void)requestOneTimePayment:(BTPayPalCheckoutRequest *)request
 
 #pragma mark - Helpers
 
+- (void)startPayPalNativeCheckout:(BTPayPalCheckoutRequest *)request
+                    configuration:(BTConfiguration *)configuration
+                        pairingId:(NSString *)payToken
+                       completion:(void (^)(BTPayPalAccountNonce *, NSError *))completionBlock {
+
+    NSString *payPalClientID = [configuration.json[@"paypal"][@"clientId"] asString];
+
+    if (!payPalClientID) {
+        NSError *error = [NSError errorWithDomain:BTPayPalDriverErrorDomain
+                                             code:BTPayPalDriverErrorTypeIntegration
+                                         userInfo:@{NSLocalizedDescriptionKey: @"PayPal Native Checkout failed because an empty or nil Client ID was retrieved from the configuration."}];
+        completionBlock(nil, error);
+        return;
+    }
+
+    NSInteger environment;
+    if ([configuration.environment isEqualToString: PayPalEnvironmentProduction]) {
```

Comment:
The constant for `PayPalEnvironmentProduction` is `"live"`, but the config returns `"production"`. 

Looks like `PayPalEnvironmentProduction` isn't used anywhere else in the SDK - so we can update it to `PayPalEnvironmentProduction = @"production"`.